applications/IgaApplication/custom_utilities/create_breps_sbm_utilities.h

```diff
@@ -232,7 +232,7 @@ class CreateBrepsSbmUtilities : public IO
         if (rSurrogateModelPartOuter.NumberOfConditions() > 0)
         {
             for (auto &i_cond : rSurrogateModelPartOuter.Conditions()) {
-                Geometry<NodeType>::PointsArrayType points;
+                Geometry<Node>::PointsArrayType points;

```

Comment:
```suggestion
                Geometry<TNodeType>::PointsArrayType points;
```
Shouldn't it be like this?